packages/entity-server/src/routes/hierarchy/relations/RelationsResponseBuilder.ts

```diff
@@ -48,7 +48,9 @@ export class RelationsResponseBuilder {
       hierarchyId,
       ancestorType,
     );
-    const ancestorCodes = Object.values(descendantAncestorMapping).map(ancestor => ancestor.code);
+    const ancestorCodes = [
+      ...new Set(Object.values(descendantAncestorMapping).map(ancestor => ancestor.code)),
+    ];
```

Comment:
This fixed a small issue (wasn't causing any bugs) where the requested ancestors would be something like : `WHERE code in ('TO','TO','TO','TO',etc..)`